src/server/world/terrain_generation.cpp

```diff
@@ -60,7 +61,9 @@ float getNoiseAt(const glm::vec2 &blockPosition, const glm::vec2 &chunkPosition,
         float x = voxelX * frequency / options.smoothness;
         float y = voxelZ * frequency / options.smoothness;
 
-        float noise = glm::simplex(glm::vec3{x, y, seed});
+        float seed_float;
+        std::memcpy(&seed_float, &seed, sizeof(float));
```

Comment:
To be nit-picking, `sizeof(int) >= 2` and `sizeof(float) == 4` so it can incoke undefined behaviour on some platform
I would advice using `std::min(sizeof(int), sizeof(float))` as the third argument, and possibly creating it's own function..

Reply:
I think using uint32_t will work best, as I intended a seed to be 32 bits long.